web/modules/custom/helloworld/src/Form/HelloworldSettingsForm.php

```diff
@@ -0,0 +1,111 @@
+<?php 
+
+namespace Drupal\helloworld\Form;
+
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+
+  /**
+   * Class to handle the configuration form of the helloworld custom module.
+   */
+class HelloworldSettingsForm extends ConfigFormBase {
+
+  /**
+   * Function to get the form id of the configure form.
+   * @return string
+   */
+  public function getFormId() {
+    return 'helloworld_admin_settings';
+  }
+
+  /**
+   * Returns an array of the configuration fields of the form.
+   */
+  protected function getEditableConfigNames() {
+    return [
+      'helloworld.admin_settings',
+    ];
+  }
+
+  /**
+   * Function responsible for building the form elements.
+   *
+   * @param array $form
+   * @param FormStateInterface $form_state
```

Comment:
Please check any core files to know how function docblock is written properly.
Please ensure the same is fixed for all other docblocks in this PR.

Reply:
This is still not fixed.